test/models/site_updates/new_concept_update_test.rb

```diff
@@ -48,7 +48,8 @@ class SiteUpdates::NewConceptUpdateTest < ActiveSupport::TestCase
       [author].map do |maker|
         {
           handle: maker.handle,
-          avatar_url: maker.avatar_url
+          avatar_url: maker.avatar_url,
+          flair: maker&.flair
```

Comment:
This also applies to the other occurences in this file

```suggestion
          flair: maker.flair
```